includes/class-wc-payments-db.php

```diff
@@ -30,6 +30,38 @@ public function order_from_charge_id( $charge_id ) {
 		return false;
 	}
 
+	/**
+	 * Retrieve orders from the DB using a list of corresponding Stripe charge IDs.
+	 *
+	 * @param array $charge_ids List of charge IDs corresponding to an order ID.
+	 *
+	 * @return boolean|WC_Order|WC_Order_Refund
+	 */
+	public function orders_with_charge_id_from_charge_ids( $charge_ids ) {
+		global $wpdb;
+
+		$charge_id_placeholder = join( ',', array_fill( 0, count( $charge_ids ), '%s' ) );
```

Comment:
`join` -> `implode`. `join` is an alias for `implode`, so let's use the original one.